File: C program/New Batch/PrimeNumber.cpp
#include<stdio.h>
int main(){
    int num;
    printf("Enter a number: ");
    scanf("%d",&num);
    if(num==1){
        printf("Non prime");
        return 0;
    }
    int i;
    for(i=2;i<num;i++){
        if(num%i==0){
            printf("Non prime");
            break;
        }
    }
    if(i==num){
        printf("Prime");
    }
    return 0;
}

File: Java Codes/New Batch/PrimeNumber.java
import java.lang.Math;
public class PrimeNumber {
    int n;
    System.out.println("Enter a number: ");
    Scanner sc = new Scanner(System.in);
    n = sc.nextInt();
    int i;
    for(i=2;i<=(int)(Math.sqrt(n));i++){
        if(num%i==0){
            break;
        }
    }
    if(i>(int)(Math.sqrt(n))){
        System.out.println("Prime number");
    }
    else{
        System.out.println("Not a prime number");
    }
}
